PostHog/PostHogSDK.swift

```diff
@@ -315,7 +372,12 @@ let maxRetryDelay = 30.0
         ))
     }
 
-    @objc public func alias(_ alias: String, properties: [String: Any]? = nil) {
+    @objc public func alias(_ alias: String) {
+        self.alias(alias, properties: nil)
+    }
+
+    @objc(aliasWithAlias:properties:)
```

Comment:
`aliasWithAlias` is repeating because its the method name and param, but I don't have a better idea?